File: DependencyInjection/DependencyInjection/_1_ControlFreak/InMessages/NullMessage.cs
using DependencyInjection._1_ControlFreak.Interfaces;
using DependencyInjection._1_ControlFreak.Services;

namespace DependencyInjection._1_ControlFreak.InMessages
{
  class NullMessage : AcmeMessage
  {
    public void AuthorizeUsing(ActiveDirectoryBasedAuthorization authorizationRules)
    {
      
    }

    public void WriteTo(DataDestination dataDestination)
    {
       
    }
  }
}

File: Java/dependency-injection/control-freak/src/main/java/com/github/grzesiek_galezowski/DependencyInjection/_1_ControlFreak/InMessages/NullMessage.java
package com.github.grzesiek_galezowski.DependencyInjection._1_ControlFreak.InMessages;

import com.github.grzesiek_galezowski.DependencyInjection._1_ControlFreak.Interfaces.AcmeMessage;
import com.github.grzesiek_galezowski.DependencyInjection._1_ControlFreak.Interfaces.DataDestination;
import com.github.grzesiek_galezowski.DependencyInjection._1_ControlFreak.Services.ActiveDirectoryBasedAuthorization;

public class NullMessage implements AcmeMessage {
    public void authorizeUsing(ActiveDirectoryBasedAuthorization authorizationRules) {

    }

    public void writeTo(DataDestination dataDestination) {

    }
  }
